gst/nnstreamer/tensor_decoder/tensordec.c

```diff
@@ -675,12 +676,19 @@ gst_tensordec_transform (GstBaseTransform * trans,
     guint i, num_tensors;
 
     num_tensors = self->tensor_config.info.num_tensors;
+    /** Internal logic error. Negotation process should prevent this! */
     g_assert (gst_buffer_n_memory (inbuf) == num_tensors);
 
     for (i = 0; i < num_tensors; i++) {
       in_mem[i] = gst_buffer_peek_memory (inbuf, i);
-      status = gst_memory_map (in_mem[i], &in_info[i], GST_MAP_READ);
-      g_assert (status);
+      if (FALSE == gst_memory_map (in_mem[i], &in_info[i], GST_MAP_READ)) {
+        guint j;
+        ml_logf ("Failed to map in_mem[%u].\n", i);
+
+        for (j = i - 1; j >= 0; j--)
```

Comment:
Q. I think `i` can be 0. Would this have problem because of overflow of uint?

Reply:
Uh.. yes.. j is unsigned. Thus, this incurs infinite loop. I'll reverse it (```for (j = 0; j < i; j++)```)
Ther are a lot of similar codes to be fixed, too.